app/models/user.rb

```diff
@@ -1,7 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
   validates :name, presence: true
-  validates :photo, presence: true
-  validates :post_counter, presence: false, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :email, presence: true
+  validates :photo, presence: false
+  validates :post_counter, presence: false
+  # numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] It would be nice if you could remove all commented unused code from your codebase as that makes your codebase not clean and professional. Let's try to have a clean and professional codebase and by so doing, we remove all commented unused codes from our codebase 👍🏽